provider/marathon/marathon.go

```diff
@@ -369,6 +449,62 @@ func (p *Provider) hasCircuitBreakerLabels(application marathon.Application) boo
 	return ok
 }
 
+// Map of services properties
+// we can get it with label[serviceName][propertyName] and we got the propertyValue
+type servicePropertyValues map[string]string
+
+type labelServiceProperties map[string]servicePropertyValues
+
+// Check if for the given application, we find labels that are defining services
+func (p *Provider) hasServices(application marathon.Application) bool {
```

Comment:
Since we are actually using it internally in the build from this PR, should I have another PR created just asking for this method, or can we keep it here?  It would be similar to #1693 `getSubDomain` one...